examples/csharpx/hellowithdeps.csx

```diff
@@ -0,0 +1,14 @@
+#r "/kubeless/packages/newtonsoft.json/11.0.2/lib/netstandard2.0/Newtonsoft.Json.dll"
```

Comment:
We could ask the user to reference just the dll name and we could perform the whole path. What do you think?